src/test/resources/features/ListDatasets.feature

```diff
@@ -9,6 +9,7 @@ Feature: List Datasets order by Title
     And I login as "user" with password "password"
     When I list the dataset
     Then The response code is 200
+    And User "owner" owns 1 dataset
```

Comment:
No need to check this because ownership does not change due to this feature. You can also remove the "I login as..." step (or you can remove it in some of the scenarios) to show that authentication is not required to list datasets